### DISCUSSION TITLE
Does RmlUi support rlottie dynamic properties?

### DISCUSSION THREAD

**Discussion by vanlueckn** [mikke89/RmlUi#324] [Q&A] [answered]
rlottie does support dynamic properties to update parts of lottie animations at runtime. Does RmlUi support this behaviour?

**Comment by mikke89** [comment, marked as answer]
Hi.

The rlottie plugin is very much barebones. Whichever features are supported by rlottie should be easy enough to wrap into the interface for the rlottie element. I suggest that you start here: https://github.com/mikke89/RmlUi/blob/master/Source/Lottie/ElementLottie.cpp. Then you can either add your desired features as a new functions, by responding to attributes, or perhaps retrieving the rlottie handle directly, whichever makes the most sense for any particular feature.

If your changes are something that could be useful to others then please make a pull request as well ;)
